src/test/java/io/confluent/examples/streams/utils/PairOfDoubleAndLongDeserializer.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright Confluent Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.confluent.examples.streams.utils;
+
+import org.apache.kafka.common.serialization.Deserializer;
+import org.apache.kafka.common.serialization.DoubleDeserializer;
+import org.apache.kafka.common.serialization.LongDeserializer;
+
+import java.util.Map;
+
+public class PairOfDoubleAndLongDeserializer implements Deserializer<Pair<Double, Long>> {
```

Comment:
This class is required only for allowing the Kafka consumer client to read the test app's output data from Kafka for validation purposes. Unfortunately, I am not aware of a way to pass a parameterized/configured `PairDeserializer<Double, Long>` directly to the consumer via `ConsumerConfig.VALUE_DESERIALIZER_CLASS_CONFIG`.